server/cmwell-ws/src/pack/logback.xml

```diff
@@ -32,6 +32,45 @@
         <appender-ref ref="FILE" />
         <maxFlushTime>0</maxFlushTime>
         <discardingThreshold>0</discardingThreshold>
+        <filter class="ch.qos.logback.core.filter.EvaluatorFilter">
+            <evaluator class="ch.qos.logback.classic.boolex.OnMarkerEvaluator">
+                <marker>markedMsg</marker>
+            </evaluator>
+            <onMismatch>NEUTRAL</onMismatch>
+            <onMatch>DENY</onMatch>
+        </filter>
+    </appender>
+
+    <appender name="FILE_MARKER" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>${application.home:-.}/logs/markedLog.log</file>
```

Comment:
either change to marked or some other name that should match the fileNamePattern below